### PR TITLE
Add an acceptance test, add creating a snapshot from the target table

### PR DESCRIPTION
**Pull request by voscob**
### Description

I've added an acceptance test, added a snapshot to use it instead of the target table in snapshot mode, and update the source integration test with deleting the tracking table for the test.

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/ConduitIO/conduit-connector-oracle/pulls) for the same update/change.
- [ ] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,31 +17,32 @@ Run `make build`.
...
+Run `make test` to run all unit and integration tests, as well as an acceptance test. To pass the integration and
+acceptance tests, set the Oracle database URL to the environment variables as an `ORACLE_URL`.
 
 ## Source
 
 The Oracle Source connects to the database using the provided `url` and starts creating records for each table row and
-each change detected. The first time Source runs, it creates an additional table and a trigger to track changes in
-the `table` and launches Snapshot mode. Then, when all the records have been read, Source switches to CDC mode. More
-information [inside the Change Data Capture section](#change-data-capture).
+each detected change. The first time Source runs, it makes a snapshot, creates a tracking table, and a trigger to track
```
Comment (maksenius) [inline README.md:26]: please add link from oracle to snaphsot for understanding what is it

File: source/iterator/iterator.go [hunk 2]
```diff
@@ -76,6 +73,14 @@ func New(ctx context.Context, params Params) (*Iterator, error) {
 		batchSize:      params.BatchSize,
 	}
 
+	// hash the table name to use it in tracking table and snapshot
```
Comment (maksenius) [inline source/iterator/iterator.go:76]: please add comment why we do it

File: README.md (hunk 1 above)
Comment (voscob) [reply README.md]: Done.

File: source/iterator/iterator.go (hunk 2 above)
Comment (voscob) [reply source/iterator/iterator.go]: Done.

**Pull request merged**